.github/workflows/labeler.yml

```diff
@@ -25,6 +28,61 @@ jobs:
     steps:
       # Extract type and try to add it as a label
     - run: gh pr edit "$PR_NUMBER" --add-label "$(echo "$PR_TITLE" | sed -E 's|([[:alpha:]]+)(\(.*\))?!?:.*|\1|')" || true
-
       # Extract scope and try to add it as a label
     - run: gh pr edit "$PR_NUMBER" --add-label "$(echo "$PR_TITLE" | sed -E 's|[[:alpha:]]+\((.+)\)!?:.*|\1|')" || true
+
+  add-reviewer:
+    runs-on: ubuntu-latest
+    needs: ["triage", "type-scope"]
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GH_REPO: ${{ github.repository }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
+
+    steps:
+      - name: Get labels
+        id: labels
+        run: echo "::set-output name=labels::$(gh pr view $PR_NUMBER --json labels --jq '.labels.[].name' | tr '\n' '%')"
+
+      # The % at the end of the label is a hack to avoid selecting a label that
+      # is substring of another label. So if there is a label "cinema" then we
+      # don't accidentally want to interpret that as the label "ci"
+
+      - if: contains(steps.labels.outputs.labels, 'api%')
+        run: gh pr edit $PR_NUMBER --add-reviewer muniter
+
+      - if: contains(steps.labels.outputs.labels, 'ci%')
+        run: gh pr edit $PR_NUMBER --add-reviewer jamessan
+
+      - if: contains(steps.labels.outputs.labels, 'diagnostic%')
+        run: gh pr edit $PR_NUMBER --add-reviewer gpanders
+
+      - if: contains(steps.labels.outputs.labels, 'documentation%')
+        run: gh pr edit $PR_NUMBER --add-reviewer clason
+
+      - if: contains(steps.labels.outputs.labels, 'extmarks%')
+        run: gh pr edit $PR_NUMBER --add-reviewer muniter
+
+      - if: contains(steps.labels.outputs.labels, 'gui%')
+        run: gh pr edit $PR_NUMBER --add-reviewer smolck
+
+      - if: contains(steps.labels.outputs.labels, 'lsp%')
+        run: gh pr edit $PR_NUMBER --add-reviewer mjlbach,mfussenegger
+
+      - if: contains(steps.labels.outputs.labels, 'lua%')
+        run: gh pr edit $PR_NUMBER --add-reviewer gpanders
+
+      - if: contains(steps.labels.outputs.labels, 'runtime%')
+        run: gh pr edit $PR_NUMBER --add-reviewer seandewar
```

Comment:
Are you sure about this? I think the `runtime` label is more likely applied to a PR like #16930.

Reply:
runtime == funtime :)

(No action by assigned people is _required_; this is is more intended as a service for team members to allow them to stop subscribing to _every_ issue and still get notified of things potentially of interest to them.)

Reply:
OK. I'll remove it.

Reply:
@zeertzjq also, are there any labels you want to automatically be a reviewer for?

Reply:
No.

Reply:
You sound a bit hesitant. Just let me know if you change your mind.